### ISSUE TITLE
Even if  I use another  initialPort , the 7000 7001 7002 port  still used

### ISSUE THREAD

**Issue by aristotll** [redis-developer/testcontainers-redis#4]
<img width="538" alt="image" src="https://user-images.githubusercontent.com/13341544/155454057-9755874b-f6ff-4bf2-955f-8b609240dd36.png">
<img width="1083" alt="image" src="https://user-images.githubusercontent.com/13341544/155454068-1aab0b73-9634-4ab4-ae06-ad27c2c13ab3.png">

The update method called in constructor is not suitable.

**Comment by ash-sykes-acn** [comment]
Agreed. Although I can assign more ports via `withInitialPort` the default ports are still being added (which is problematic as I'm using the)
